pipelines/upgrade/vars/base.yml

```diff
@@ -1,3 +1,4 @@
+forklift_directory: "{{ playbook_dir | dirname | dirname }}"
```

Comment:
Why is dirname listed twice here?

Reply:
`playbook_dir` is `pipelines/upgrade` so the forklift root is `pipelines/upgrade/../..`